config/crd/bases/toolchain.dev.openshift.com_toolchainconfigs.yaml

```diff
@@ -47,42 +47,20 @@ spec:
                   automaticApproval:
                     description: Keeps parameters necessary for automatic approval
                     properties:
+                      domains:
+                        description: 'Comma-separated email domains to consider for
+                          auto-approval. For example: "domain.com,anotherdomain.org"
+                          If domains is not set and enabled is true, it will default
+                          to auto approving all authenticated emails. If domains is
+                          set and enabled is true, it will allow auto approving only
+                          for authenticated emails under the domains entered. If enabled
+                          is false domains will be ignored.'
+                        type: string
                       enabled:
                         description: Defines if the automatic approval is enabled
                           or not
                         type: boolean
                     type: object
-                  capacityThresholds:
-                    description: 'Keeps parameters necessary for configuring capacity
-                      limits Deprecated: This is no longer used for anything.'
-                    properties:
-                      maxNumberOfSpacesPerMemberCluster:
-                        additionalProperties:
-                          type: integer
-                        description: 'Contains a map of maximal number of spaces that
-                          can be provisioned per member cluster mapped by the cluster
-                          name Deprecated: This is no longer used for anything.'
-                        type: object
-                        x-kubernetes-map-type: atomic
-                      resourceCapacityThreshold:
-                        description: 'Contains capacity threshold configuration Deprecated:
-                          This is no longer used for anything.'
-                        properties:
-                          defaultThreshold:
-                            description: It is the default capacity threshold (in
-                              percentage of usage) to be used for all member clusters
-                              if no special threshold is defined
-                            type: integer
-                          specificPerMemberCluster:
-                            additionalProperties:
-                              type: integer
-                            description: Contains a map of specific capacity thresholds
-                              (in percentage of usage) for particular member clusters
-                              mapped by their names
-                            type: object
-                            x-kubernetes-map-type: atomic
-                        type: object
-                    type: object
```

Comment:
This was picked up when I rebased and generated CRD again. Looks to related to changes in this PR: https://github.com/codeready-toolchain/api/pull/411/files

Reply:
Looks like this PR needs to be merged before this one then I can rebase: https://github.com/codeready-toolchain/host-operator/pull/1000/files